presto-native-execution/presto_cpp/presto_protocol/connector/clp/ClpConnectorProtocol.cpp

```
@@ -0,0 +1,161 @@
+#include "presto_cpp/presto_protocol/connector/clp/ClpConnectorProtocol.h"
+
+#include <cstdint>
+
+namespace facebook::presto::protocol::clp {
+
+namespace {
+
+std::string readUTF(const std::string& data, size_t& offset) {
+  VELOX_CHECK_LE(
+      offset + 2, data.size(), "Not enough bytes for UTF length");
+  uint16_t len = __builtin_bswap16(
```

Clang (14.0.6)

[warning] variable 'len' is not initialized

(cppcoreguidelines-init-variables)